peer.go

```diff
@@ -2388,10 +2388,10 @@ func (p *peer) WipeChannel(chanPoint *wire.OutPoint) error {
 // local features vectors. If feature vectors are incompatible then disconnect.
 func (p *peer) handleInitMsg(msg *lnwire.Init) error {
 	p.remoteLocalFeatures = lnwire.NewFeatureVector(
-		msg.LocalFeatures, lnwire.LocalFeatures,
+		msg.LocalFeatures, lnwire.Features,
 	)
 	p.remoteGlobalFeatures = lnwire.NewFeatureVector(
-		msg.GlobalFeatures, lnwire.GlobalFeatures,
+		msg.GlobalFeatures, lnwire.Features,
```

Comment:
Can something go wrong here with communicating to older nodes previously required local-only features as global and vv? Maybe fixed in a later commit

Reply:
it is less restrictive, so we might accept a required global feature in local and vice versa, but that only applies if someone lands on an intermediate commit.

Reply:
reworked so that we send ff first, then merge the remote's features